docs/UsersGuide/source/Introduction.rst

```diff
@@ -73,12 +73,12 @@ The UFS SRW Application has been designed so that any sufficiently up-to-date ma
 
 * POSIX-compliant UNIX-style operating system
 
-* >40 GB disk space
+* >102 GB disk space
 
-   * 18 GB input data from GFS, RAP, and HRRR for "out-of-the-box" SRW App case described in :numref:`Chapter %s <BuildRunSRW>`
-   * 6 GB for :term:`HPC-Stack` full installation
-   * 1 GB for ufs-srweather-app installation
-   * 11 GB for 48hr forecast on CONUS 25km domain
+   * 54 GB input data from GFS, RAP, and HRRR for "out-of-the-box" SRW App case described in :numref:`Chapter %s <BuildRunSRW>`
```

Comment:
If users are only using one of these (e.g., GFS data), would it still be 54 GB?

Reply:
@gspetro-NOAA - yes, it's only the fix and GFS data for the 12-hour run. The tar archive for the new release is over 53 GB (and this was before addition of the missing 12-h file, another 324MB)

Reply:
Some more changes will need to be done, after some data updates. Let me follow with this as we prepare the data for the upload!